### PR TITLE
Dev get my actual projects

### PR DESCRIPTION
**Pull request by mcgnly**

### CODE REVIEW

File: src/entities/User.js [hunk 1]
```diff
@@ -95,13 +95,48 @@ export default class User {
...
                     reject(error);
                 });
             });
+        });                        
+    }
+
+    getPublisherApps(pubsArray){
+        return new Promise((resolve, reject) => {
+            let appsArray = [];
+            pubsArray.forEach(
```
Comment (SmbatYeranyan) [inline src/entities/User.js]: Consider moving this to a more optimized loop if the endpoint returns thousands of items. forEach is heavy

File: src/entities/User.js [hunk 2]
```diff
@@ -95,13 +95,48 @@ export default class User {
 
     getMyApps() {
         return new Promise((resolve, reject) => {
+            this.getMyPublishers().then((res)=>{
+                this.getPublisherApps(res).then((res2)=>{
+                    resolve(res2);
+                }, (err)=>{reject(err)}
```
Comment (SmbatYeranyan) [inline src/entities/User.js]: ERR or Error. Lets keep a consistent form in the future.

**Comment by SmbatYeranyan** [comment]
Looks good, ready for merge 👍

**Comment by mcgnly** [comment]
What is less heavy? While? If it has to do the thing the same amount of
times and isn't keeping track of an additional counter with like a for I=
something to something else, why is one heavier than another?

**Pull request closed without merging**